### DISCUSSION TITLE
【灌水】黑曜石营的士兵疑似有点太强硬了

### DISCUSSION THREAD

**Discussion by March-777th** [share121/inter-knot#2096] [其他]
<img width="1280" alt="Snipaste_2024-08-14_17-50-10" src="https://github.com/user-attachments/assets/3c86d461-a165-4d2d-92fa-58e8f0fbc4c1">

今天约治安局的青衣长官出来喝茶，结果刚出家门就被黑曜石营的士兵拖去训练了<img src="https://bbs-static.miyoushe.com/static/2024/07/31/73346f2b98d45b62da9139d2774fa856_8641114165088315754.png" alt="三月七 告退" width="75" >我缺的青衣这一块谁给我补啊<img src="https://img-static.mihoyo.com/communityweb/upload/9939adf214795d97dad9abd90265aab2.png" alt="帕姆_哭哭" width="75" >

**Comment by FYGT27** [comment]
疑似相同待遇，不要啊我的青衣前辈！

**Comment by Soldier11-ObsidianBarracks** [comment]
你什么意思，诽谤我啊！我好心带你们去拉练
